src/components/Tooltip/index.js

```diff
@@ -12,4 +13,16 @@ Tooltip.propTypes = {
   children: React.PropTypes.node.isRequired
 }
 
-export default Tooltip
+export default styled(Tooltip)`
+  &:before {
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 0 8px 9px 8px;
+    border-color: transparent transparent #fff transparent;
+    content: '';
+    position: absolute;
+    left: 20px;
+    top: -9px;
```

Comment:
@DrShpongle any way we can avoid these hard-coded values in the future to make this more dynamic?